services.php

```diff
@@ -126,12 +126,12 @@
             );
         },
         /*
-         * The placeholder template for about and settings.
+         * The placeholder template for about page.
          *
          * @since [*next-version*]
          */
-        'eddbk_ui_coming_soon_template' => function (ContainerInterface $c) {
-            $templateFile = 'templates/about-settings-coming-soon.html';
+        'eddbk_ui_about_template' => function (ContainerInterface $c) {
```

Comment:
How about something like [`_makeTemplate()`][`WpBookingsFrontUi#_makeTemplate()`] instead?

[`WpBookingsFrontUi#_makeTemplate()`]: https://github.com/RebelCode/rcmod-wp-bookings-front-ui/blob/develop/src/WpBookingsFrontUi.php#L242

Reply:
Thought about this. The problem is that all services in this package are located in separate file, `services.php`, and services definition is the function that returns array. You sent a link on method that is a member of module class, and in that module all services are defined in the module class, so we have access to properties (`$this->templateFactory`).

So can we live this simple functions as is?